src/ODEs/ODETools/GeneralizedAlpha.jl

```diff
@@ -47,7 +47,60 @@ function solve_step!(
 
 end
 
+
+function solve_step!(
+  x1::NTuple{3,AbstractVector},
+  solver::GeneralizedAlpha,
+  op::ODEOperator,
+  x0::NTuple{3,AbstractVector},
+  t0::Real,
+  cache) # -> (uF,tF)
+
+  dt = solver.dt
+  ρ∞ = solver.ρ∞
+  αf = ρ∞/(ρ∞ + 1.0)
+  αm = (2*ρ∞ - 1.0)/(ρ∞ + 1.0)
+  γ = 0.5 - αm + αf
+  β = 0.25*((1.0 - αm + αf)^2)
+  tαf = t0 + (1.0-αf)*dt
```

Comment:
In the GeneralizedAlpha for 1st order  the parameters are defined as follows:
```
αf = 1.0/(1.0 + ρ∞)
  αm = 0.5 * (3-ρ∞) / (1+ρ∞)
  γ = 0.5 + αm - αf
  tαf = t0+αf*dt
  ```
  We should keep the same definitions.